test/lib.api.v1.report-controller.test.js

```diff
@@ -64,11 +65,11 @@ describe('Report controller', function() {
 
     it('should query and filter reports', function(done) {
       request(reportController)
-        .get('/api/v1/report?keywords=one&status=unassigned')
+        .get('/api/v1/report?keywords=one')
```

Comment:
Should add &media=foo so that we're testing the filtering on attribs other than keywords. keyword filtering is separate code so it's important to test other attribs.